src/Codeception/Lib/InnerBrowser.php

```diff
@@ -1651,7 +1648,7 @@ public function seeOptionIsSelected($selector, $optionText)
     public function dontSeeOptionIsSelected($selector, $optionText)
```

Comment:
$optionText is string

Reply:
this signature comes from [`src/Codeception/Lib/Interfaces/Web.php`](https://github.com/Codeception/Codeception/blob/60c9ed971d4bae7328dde7fd44829b1f849403ab/src/Codeception/Lib/Interfaces/Web.php#L848)

Reply:
Please update the interface first. Half-done feature of chainable methods is worse than no such feature.
Please update composer.json of dependent modules to require codeception 5.0

Reply:
OK.

Since the above update will require Codeception 5 I will also update the PHP version to 7.3 instead of 7.1 in this PR and those of the other dependent modules that I promised to update.

I will try to do it this week.

Reply:
@Naktibalda Done.